codes.md

```diff
@@ -76,11 +94,12 @@ Codes
 ## FOREIGN_TERMINOLOGY. Even very technical people find some data abstraction concepts, language foreign
   - One Supercomputing survey participant was confused by "item" vs "attribute" terminology. In their response, "tens of items" was selected (initially thinking about table columns), but then selected millions after discussing with A-1, A-3
   - Other generalizations: "A spreadsheet," even though "Tabular" data was an option and we gave them the opportunity to rename it (although they may not have known)!
+  - “I had a hard time, actually, thinking of the dataset as tabular in a way that I had not previously thought of it as tabular (which I believe was the intent of this survey). The questions did not seem to specifically account for this possibility as distinct from considering it as tabular for the first time.” (from Alternative Dataset Debrief) -- A-3: This may be an indicator that the language use is founded in tables, already doesn't fit some datasets
```

Comment:
This one is really interesting—this definitely fits as evidence for this code, but the "first time" angle seems like there might be another one, especially in light of the conversation we just had on Slack. I may reuse this elsewhere...